src/pip/_internal/commands/uninstall.py

```diff
@@ -64,6 +68,12 @@ def run(self, options, args):
             )
             if req.name:
                 reqs_to_uninstall[canonicalize_name(req.name)] = req
+            else:
+                logger.warning(
+                    "Invalid requirement: '%s' ignored -"
+                    " the uninstall command expects named"
+                    " requirements/requirements file.", name
```

Comment:
```suggestion
                    " requirements.", name
```
I don't think we handle requirements file directly.